cmd/keytransparency-client/grpcc/grpc_client.go

```diff
@@ -26,6 +26,7 @@ import (
 	"log"
 	"time"
 
+	"github.com/gogo/protobuf/proto"
```

Comment:
What's special about gogo's proto?
I'm pretty sure the github.com/golang/protobuf/proto has Equal() that should do what you want?

Reply:
goimport automatically includes this.  I'll delete it off my machine. We don't want to use it.